sdk/communication/azure-communication-rooms/tests/test_rooms_client_async.py

```diff
@@ -205,6 +215,24 @@ async def test_update_room_exceed_max_timerange_async(self):
             valid_from =  datetime.now() + timedelta(days=3)
             valid_until =  datetime.now() + timedelta(weeks=29)
 
+            with pytest.raises(HttpResponseError) as ex:
+                await self.rooms_client.update_room(room_id=create_response.id, valid_from=valid_from, valid_until=valid_until)
+
+                # delete created room
+                await self.rooms_client.delete_room(room_id=create_response.id)
+
+                assert str(ex.value.status_code) == "400"
+                assert ex.value.message is not None
+    @recorded_by_proxy_async
```

Comment:
Extreme nit: you can add a line break here

Reply:
Sounds good, will add it!